src/test/java/com/google/sps/servlets/PotentialMatchesServletTest.java

```diff
@@ -186,9 +207,21 @@ public void fourUsersOneMutualConnection() throws Exception {
     addTestUserEntityToDatastore(datastore, TEST_USER_4_ID, TEST_USER_4_NAME,
       TEST_USER_4_EMAIL, TEST_USER_4_BIO, testUser4FriendsList);
 
-    String actualOutput = execute(TEST_USER_4_ID);
+    String actualOutput_1 = execute(TEST_USER_4_ID);
+    String expected_2 = actualOutput_1.equals(TEST_USER_2_ID) ? TEST_USER_3_ID : TEST_USER_2_ID;
```

Comment:
nit: why is this expected_2? I don't see an expected_1

Reply:
I meant to have this as the expected output for the second call to the servlet's doGet method. I can change this variable to be named expectedSecondOutput to clarify what it's for!